### PR TITLE
Issue/3941 stats jetpack missing prompt

### LINKED ISSUE #3941: Stats: Incorrectly prompted that Jetpack was not installed.
My self-hosted test blog has the latest version of the Jetpack (3.9.6) plugin installed, but it is not connected to WordPress.com.
When tapping the Stats option in the blog details screen, I see the stats activity appear and try to sync. Eventually it stops syncing and I'm prompted with an error that reads: 
"Jetpack plugin not found, The Jetpack plugin is required for stats. Do you want to install Jetpack? No / Yes" 
If I tap yes I'm taken to the Add Plugins page and I see in the list of plugins that Jetpack is *already* installed.

### PR DESCRIPTION
**Pull request by daniloercoli**
Fixes #3941 by checking both `jetpack_version`, and `jetpack_blog_id` before showing the prompt to the user. 

Needs review: @aerych - Please double check the wording of the new messages ;)

Note 1: There are still small issues if you disable the plugin on the server after a successful connection to Stats from Mobile. (Options are updated with a small delay on Android).

Note 2: I see problem connecting to WordPress.com with an account != from the main one. It seems to be a problem somewhere in the networking layer that lost the correct access token. I will open a new ticket for it.

### CODE REVIEW

**Comment by daniloercoli** [comment]
@astralbodies I've added 1 new Analytics event in this PR. Already added to the doc.

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1042,9 +1042,15 @@
...
     <string name="passcodelock_hint"></string>
+
+    <!--
+      Jetpack strings
+    -->
     <string name="jetpack_message">The Jetpack plugin is required for stats. Do you want to install Jetpack?</string>
     <string name="jetpack_message_not_admin">The Jetpack plugin is required for stats. Contact the site administrator.</string>
     <string name="jetpack_not_found">Jetpack plugin not found</string>
+    <string name="jetpack_not_connected">Jetpack plugin not connected</string>
+    <string name="jetpack_not_connected_message">The Jetpack plugin is installed, but not connected with WordPress.com. Do you want to connect Jetpack?</string>
```
Comment (aerych) [inline WordPress/src/main/res/values/strings.xml]: `but not connected with WordPress.com` should probably be `but not connected to WordPress.com`. Better preposition and to match the connect button in the admin. 
screen shot 2016-04-19 at 3 56 45 pm

**Comment by aerych** [comment]
Changes are super cool @daniloercoli :) 
One small grammer nit pick but otherwise :shipit from me.

**Comment by aerych** [comment]
Hiya @maxme.  For paranoia sake, would you mind taking a peek at the code as well?

File: WordPress/src/main/res/values/strings.xml (hunk 1 above)
Comment (daniloercoli) [inline WordPress/src/main/res/values/strings.xml]: Fixed!

**Comment by maxme** [comment]
Looks good! :shipit:

**Pull request merged**